docs/Home.md

```diff
@@ -2,41 +2,77 @@
+  
+Note: While the Quarkus project has the "keys" to publish under `io.quarkiverse`, each project in Quarkiverse is driven and maintained by 
+the lead of that project. They decide what happens with the project - Quarkiverse will just expect releases to be compatible with Quarkus - if a project
+for some reason cannot do a release we can in the best scenario help or in the worst scenario we'll temporarily exclude the project from the registry to avoid
+end-users to have a bad experience.
+ 
+If you do not want to commit to the above guidelnes then you are more than welcome to publish an extension outside of Quarkus or Quarkiverse; and then you can
+still be listed in the registry - it is just not handled automtically. In this case we do please request you use your own group id to clearly identify those artifacts are from a third-party.
+
 # Joining Quarkiverse
 
 Projects hosted in the Quarkiverse organization on github should respect the following conventions and policies.
 
+## Identify Project Maintainer ##
+
+Each project will have a github team, and in here at least one must be listed and active as maintainer. This is the person that will be expected to lead and drive the project.
+
 ## Repository name
 
-The repository name under the quarkiverse organization should have `quarkiverse-` prefix. This will help avoid potential conflicts when cloning and/or forking the repository.
+The repository name under the quarkiverse organization should be `quarkiverse-<project-name>`. This will help avoid potential conflicts when cloning and/or forking the repository.
 
 ## Project's Maven artifact `groupId`s
 
 The Maven `groupId` of the project's artifacts should follow the following format: `io.quarkiverse.<project-name>`. In other words, the project’s `groupId` should start with the `io.quarkiverse.` prefix followed by the project-specific unique name in which dots aren't allowed.
 
 ## Root package name
 
-The root package name is expected to be the same as the project’s Maven artifact `groupId`.
+The root package name is expected to be the same as the project’s Maven artifact `groupId`, i.e. `io.quarkiverse.jberet`, if project name is `jberet`.
 
 ## Parent POM
 
 All Quarkiverse projects are expected to use [io.quarkiverse:quarkiverse-parent](https://github.com/quarkiverse/quarkiverse-parent) as the parent POM.
 This POM contains common project release and artifact publishing configuration.
 
+## Provide mechanism to do cross-testing of named quarkus core and platform versions
+
+All Quarkiverse projects should have a build and test suite that can be parameterized to use specific Quarkus and Platform versions to ensure we can get
+early warning of issues of master builds; but also to verify compatibiltiy with maintanence branch/tags.
+
+Mechanism is To be defined, but something like having a script that will take quarkus core and quarkus platform Group, Artifact and IDs.
```

Comment:
THE mechanism. I would also clarify that the typical platform would be io.quarkus:quarkus-bom. I think people may be confused what a platform is.

Reply:
I would expect it to be universal platform, not just the core.

Reply:
I wouldn't pick the universal one but the minimal one. Simply because some extensions could be developed as alternatives or be for some other good reason incompatible.

Reply:
We are talking about cross testing here - not about the extensions own build.

This is about identifying possible conflicts/issues.

universal platform seems to be a good fit for that afaics and makes the extension verified to work with that platform.

We can *add* another cross-test for core if that helps ?

Reply:
Right. Let's identify a couple of testing points here. An extension will be using some platform for its built. So, when a new version of that platform is released it makes sense to test the extension against the new version of the platform the extension is using in its own build.
Then there is an option to try and test the extension with some other platform.
I was just saying that a typical extension project will most probably be based on quarkus-bom.